src/explore-education-statistics-admin/src/pages/publication/PublicationPageContainer.tsx

```diff
@@ -55,7 +55,7 @@ const PublicationPageContainer = ({
     isLoading: loadingPublication,
     retry: reloadPublication,
   } = useAsyncHandledRetry(() =>
-    publicationService.getMyPublication(publicationId),
+    publicationService.getPublication(publicationId, true),
```

Comment:
As mentioned in `publicationService.ts`, we should add a generic type parameter so that we can call this like:
```ts
publicationService.getPublication<PublicationWithPermissions>(publicationId, true)
```